Documentation/Signoffs/jdjoyce42-power-signoff.md

```diff
@@ -0,0 +1,79 @@
+# Power Subsystem
+
+
+  
+ 
+## Function of the Subsystem
```

Comment:
The statement of function does not need to be this long. Tell what it is to do. Remove the fluff.